Makefile

```diff
@@ -0,0 +1,8 @@
+
+all: runKompile runJavac
+
+runKompile: semantics/llvm-syntax.k semantics/llvm-syntax-kompiled
+	kompile semantics/llvm-syntax.k --backend java
```

Comment:
Why do you need this?  `--backend java` is now the default option in K.  Why not just call `compile` as we usually do?